crates/router/src/core/payments/retry.rs

```diff
@@ -571,6 +571,8 @@ pub fn make_new_payment_attempt(
         customer_acceptance: Default::default(),
         profile_id: old_payment_attempt.profile_id,
         organization_id: old_payment_attempt.organization_id,
+        shipping_cost: old_payment_attempt.shipping_cost,
+        order_tax_amount: old_payment_attempt.order_tax_amount,
```

Comment:
This should be set to None right? This will be dependent on the payment method that will be chosen. This would be updated later in the update trackers